assets/src/components/p4_button/edit.js

```diff
@@ -57,31 +75,6 @@ const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
 } );
 
 const NEW_TAB_REL = 'noreferrer noopener';
-const MIN_BORDER_RADIUS_VALUE = 0;
```

Comment:
Maybe it's better to keep this part in, as we already removed its usage. Keeping it will make it a lot easier to keep this file up to date with whatever changes happen in Gutenberg.